src/Util/ClassSourceManipulator.php

```diff
@@ -121,13 +121,23 @@ public function addEntityField(ClassProperty $mapping): void
 
         $defaultValue = null;
         $commentLines = [];
-        if ('array' === $typeHint && !$nullable) {
-            $defaultValue = new Node\Expr\Array_([], ['kind' => Node\Expr\Array_::KIND_SHORT]);
-            if (null !== $mapping->enumType) {
+
+        if (null !== $mapping->enumType) {
+            if ('array' === $typeHint) {
+                // still need to add the use statement
+                $this->addUseStatementIfNecessary($mapping->enumType);
```

Comment:
I think we can eliminate the `else` conditional in this `if` statement by moving `addUserStatement...()` outside and after `if ('array' === $typeHint) is called. 

We're calling the add use statement method regardless if `array` === `$typeHint` correct?

Reply:
> We're calling the add use statement method regardless if `array` === `$typeHint` correct?

We're calling it in both cases, but if array === $typeHint, we don't want the returned string; it's just to automate the use statement.